docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -155,49 +168,9 @@ Preparing For a Release
 #. Update the staging documentation.
    (For a documentation release ``${VERSION}`` should be the base release version in this step).
 
-   .. TODO: The following steps should be automated
-
-   #. Copy release documentation from ``clusterhq-dev-docs`` to ``clusterhq-staging-docs``.
-
-      .. prompt:: bash $
-
-         gsutil -m rsync -d -r s3://clusterhq-dev-docs/$(python setup.py --version)/ s3://clusterhq-staging-docs/en/${VERSION}/
-
-   #. Update redirects to point to new documentation.
-
-      .. warning:: Skip this step for weekly releases and pre-releases.
-
-      .. prompt:: bash $
-
-         gsutil -h x-amz-website-redirect-location:/en/${VERSION} setmeta s3://clusterhq-staging-docs/en/index.html
-         gsutil -h x-amz-website-redirect-location:/en/${VERSION} setmeta s3://clusterhq-staging-docs/index.html
-
-   #. Update the redirect rules in `S3`_ to point to the new release.
-
-      In the properties of the ``clusterhq-staging-docs`` bucket under static website hosting,
-      update the redirect for ``en/latest`` (for a marketing release) or ``en/devel`` to point at the new release.
-      Update the ``RoutingRule`` block matching the appropriate key prefix, leaving other ``RoutingRule``\ s unchanged.
-
-      .. code-block:: xml
-
-         <RoutingRule>
-           <Condition>
-             <KeyPrefixEquals>en/latest/</KeyPrefixEquals>
-           </Condition>
-           <Redirect>
-             <ReplaceKeyPrefixWith>en/${VERSION}/</ReplaceKeyPrefixWith>
-             <HttpRedirectCode>302</HttpRedirectCode>
-           </Redirect>
-         </RoutingRule>
+   .. prompt:: bash $
 
-   #. Create an invalidation for the following paths in `CloudFront`_, for the ``docs.staging.clusterhq.com`` distribution::
-
-         /
-         /index.html
-         /en/
-         /en/index.html
-         /en/latest/*
-         /en/devel/*
+      admin/publish-docs --doc-version ${VERSION}
```

Comment:
Python dependencies are needed to run this so the engineer should be put into a new virtualenv.